reports/pin/_comp.qmd

```diff
@@ -0,0 +1,100 @@
+{{< include ../_setup.qmd >}}
+
+## Comps
+
+This map shows the target parcel alongside the 20 most similar parcels, where
+similarity is determined by the number of matching leaf node assignments that
+the model gives to each parcel weighted by the relative importance of each tree.
+See [this
+vignette](https://ccao-data.github.io/lightsnip/articles/finding-comps.html)
+for more background on the similarity algorithm.
+
+```{r _comp_map}
```

Comment:
The code in this block is very similar to the code snippet that Dan sent me. Sorry Dan! I know you were hoping I would improve it, but it seems like a fine start to me and my understanding is that we'll be finessing this map soon anyway.

Reply:
I can finesse it a bit once it's merged or we can have @julia-klauss work on it!